.clang-format-ignore

```diff
@@ -2,3 +2,4 @@
 **/*.inc
 **/*.json
 lib/
+tools/nitrogfx/
```

Comment:
if you're going to make clang-format ignore the nirtogfx space would you also want to revert clang-format's changes to the folder?